### ISSUE TITLE
Having a corrupted wvs file throws no errors but never completes

### ISSUE THREAD

**Issue by Aaron3154518** [BoiseState-AdaptLab/adapt-lidar-tools#264]
I ran *bin/geotiff-driver -f Nayani_clipped_test.pls -e 1* and after 1 day of processing it did not finish. This file should take around 20-30 seconds to process. I am looking into why this file freezes the program.

**Comment by Aaron3154518** [comment]
I did a make geotiff-driver and it completed successfully. I will try to recreate the error and pinpoint why it wouldn't proccess.

**Comment by Aaron3154518** [comment]
I was able to recreate the bug by copying the .pls file onto the .wvs file. My reason for doing this was that a couple days ago _git status_ told me I had somehow changed Nayani_clipped_test.wvs so I reset the .wvs file. However I had already copied it into my home directory for testing and did not fix it there. Once I fixed it, the program ran as normal and once I changed the .wvs file the program failed to complete. This is definitely a bug that needs to be handled.
